src/patternfly/components/Alert/examples/Alert.md

```diff
@@ -54,7 +56,56 @@ cssPrefix: pf-c-alert
 ```
 
 ### Variations
+
 ```hbs
+<br>
+{{#> alert alert--modifier="pf-m-expandable" alert--attribute='aria-label="Expandable alert"'}}
+          {{#> alert-toggle button--attribute=(concat 'aria-labelledby="' alert--id '-toggle1 ' data-list--id '-item1" id="' alert--id '-toggle1" aria-label="Toggle details for" aria-expanded="true" aria-controls="' data-list--id '-content1"')}}
+        {{/alert-toggle}}
+  {{#> alert-icon alert-icon--type="check-circle"}}
+  {{/alert-icon}}
+  {{#> alert-title}}
+    {{#> screen-reader}}Success alert:{{/screen-reader}}
+ Success alert title
+  {{/alert-title}}
+  {{#> alert-description}}
+     Success alert description. This should tell the user more information about the alert.
+  {{/alert-description}}
+{{/alert}}
+<br>
+
+<br>
+{{#> alert alert--modifier="pf-m-expanded" alert--attribute='aria-label="Expandable alert"' alert--IsExpanded="true"
+}}
+
+        {{#> alert-toggle button--attribute=(concat 'aria-labelledby="' alert--id '-toggle1 ' data-list--id '-item1" id="' alert--id '-toggle1" aria-label="Toggle details for" aria-expanded="true" aria-controls="' data-list--id '-content1"')}}
+        {{/alert-toggle}}
+
+  {{#> alert-icon alert-icon--type="check-circle"}}
+  {{/alert-icon}}
+ {{#> alert-action-group}}
+    {{#> button button--modifier="pf-m-link pf-m-inline"}}
+      View details
+    {{/button}}
+    {{#> button button--modifier="pf-m-link pf-m-inline"}}
+      Ignore
+    {{/button}}
+  {{/alert-action-group}}
+
+  {{#> alert-title}}
+    {{#> screen-reader}}Success alert:{{/screen-reader}}
+ Success alert title
+  {{/alert-title}}
+  {{#> alert-description}}
+        This is some description for this alert that is super long. This is some description for this alert that is super long. This is some description for this alert that is long.
+        Some additional information:
```

Comment:
nit - could you turn this into a list? An easy wan to do that would be to wrap everything in a `{{#> content}}` block, then use `<p>`'s for the blocks of text, and an `<ol>` for this list - the `.pf-c-content` component will space/format everything nicely.